monkeys/helm/infinitemonkeys/templates/configmaps.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
Please introduce a values parameter, that enables or disables the monitoring integration - subsequently then creates the dashboard and podmonitors if set to "true"